scripts/ansible-ssh.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+"""
+CDT-2221 Alpha Red Team
+Author: Anthony Swierkosz; ajs2576
+
+Installs SSH keys for Ansible to use.
+"""
+
+import subprocess
+
+user = "Administrator"
+ip = "10.1.4.7"
+
+
+def windows_ssh(user, ip):
+    print(f'\n\n{"":-^48}')
+    print(f'{f" {user}@{ip} ":-^48}')
+    print(f'{"":-^48}')
+
+    subprocess.run(
+        ' '.join(['sudo', 'scp', '~/.ssh/id_rsa.pub', f'{user}@{ip}:/ProgramData/ssh/administrators_authorized_keys']),
+        shell=True)
+
+    print(f"Setting permissions for '{user}@{ip}'...")
+    subprocess.run(' '.join(['sudo', 'ssh', f'{user}@{ip}',
+                             '\'icacls.exe "C:\ProgramData\ssh\\administrators_authorized_keys" /inheritance:r /grant "Administrators:F" /grant "SYSTEM:F"\'']),
+                   shell=True)
+
+    subprocess.run(' '.join(['sudo', 'ssh', f'{user}@{ip}',
+                             '\'icacls "C:\ProgramData\ssh\\administrators_authorized_keys" /remove "NT AUTHORITY\Authenticated Users"\'']),
+                   shell=True)
+
+    subprocess.run(' '.join(['sudo', 'ssh', f'{user}@{ip}',
+                             '\'icacls "C:\ProgramData\ssh\\administrators_authorized_keys" /inheritance:r\'']),
+                   shell=True)
+
+
+def main():
+    print("Welcome to ")
```

Comment:
This doesn't do anything?

- [ ] Move everything from the interpreter module check to here. (`if __name__ == '__main__'`)
- [X] Auto option was already updated in 7471b7c47f97ceaf8f16cab024f4de32fadaa868
- [ ] Make this more robust: (Improved auto method, ability to read from file, etc.)

Reply:
Being handled by https://github.com/aisgbnok/Ansible-2221/issues/8#issue-1450977174.